index.bs

```diff
@@ -4705,14 +4705,14 @@ and |session|:
    1. Append [=CreateArrayFromList=](«|deserialized key|, |deserialized value|») to
       |deserialized key-value list|.
 
-1. Return [=success=] with data [=CreateArrayFromList=](|deserialized key-value list|).
+1. Return [=success=] with data |deserialized key-value list|.
 
 </div>
 
+Issue:  TODO distinguish between `List` and `Array`. This has been partially completed as the "value" parameter
```

Comment:
The `|value|` parameter is always an infra List, not an ECMAScript type at all, so I don't understand the updated comment.

Reply:
@jgraham Ah ok, I can remove the comment then.